docs/content/configuration/creating_pvc.md

```diff
@@ -18,12 +18,12 @@ The `persistentvolumeclaim/<pvc-name> created` message is emitted.
 
 Use the following sections, according to your PVC needs:
 
-- Creating PVC for volume with file system
-- Creating PVC for raw block volume
+- [Creating PVC for volume with `Filesystem`](#creating-pvc-for-volume-with-filesystem)
+- [Creating PVC for raw block volume](#Creating-PVC-for-raw-block-volume)
```

Comment:
why is PVC sometimes in upper case and sometimes in lower?

Reply:
it's always upper case except for in linking. as long as the links are working it doesn't make a difference.

Reply:
I understand there is no functional difference, but I think it should be consistent, especially since it is visible to the user in the browser url

Reply:
continuing [there](https://github.com/IBM/ibm-block-csi-driver/pull/415/files#r767239187)